src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -194,6 +195,51 @@ extension ExposureSubmissionCoordinator {
 		present(vc)
 	}
 
+	func showSymptomsScreen() {
+		let vc = createSymptomsViewController(
+			onPrimaryButtonTap: { [weak self] selectedSymptomsOption in
+				switch selectedSymptomsOption {
+				case .yes:
+					self?.showSymptomsOnsetScreen()
+				case .no:
+					self?.symptomsOnset = .nonSymptomatic
+					self?.showWarnOthersScreen()
+				case .preferNotToSay:
+					self?.symptomsOnset = .noInformation
+					self?.showWarnOthersScreen()
+				}
+			}
+		)
+
+		push(vc)
+	}
+
+	func showSymptomsOnsetScreen() {
+		let vc = createSymptomsOnsetViewController(
+			onPrimaryButtonTap: { [weak self] selectedSymptomsOnsetOption in
+				switch selectedSymptomsOnsetOption {
+				case .exactDate(let date):
+					guard let daysSinceOnset = Calendar.gregorian().dateComponents([.day], from: date, to: Date()).day else {
```

Comment:
Please use `Calendar.autoupdatingCurrent`. I know, we aren't covering others in the UI but for the delta calculation of days, `Calendar.autoupdatingCurrent` is valid. No need to specify a certain calendar here.

Reply:
Is the date difference in days the same in every calendar that is supported on iOS? I actually don't know this 🤷‍♂️ Since I didn't want to check and update that information once additional calendars are supported in iOS and we only support the Gregorian calendar here, I wanted to make sure the calculation is correct based on what the user selected in the interface. I would prefer to leave it as is and not take any risks here.

Reply:
I just thought of an example where it could go wrong: If the `Calendar.autoupdatingCurrent` has a leap second in the range that is calculated, while the Gregorian calendar has not, the calculation would be one day off.

Reply:
I'm still uncertain that we have to use gregorian here, but because I don't know for sure, I'm not insisting on this atm :)